packages/FormElement/stories/examples/Nested.js

```diff
@@ -1,59 +1,91 @@
+/* eslint-disable react/jsx-wrap-multilines */
 import React from "react";
-import styled from "styled-components";
 import stylers from "@paprika/stylers";
 import Input from "@paprika/input";
-import FormElement from "../../src/FormElement";
-
-const InputSet = styled.div`
-  display: flex;
-  > div {
-    margin-left: 8px;
-  }
-`;
+import FormElement from "../../src";
+import Fieldset from "../../src/components/Fieldset";
```

Comment:
To demonstrate how we would use this in the wild, we should collapse this to a single `import` like:
```js
import FormElement, { Fieldset } from "../../src";
```

Reply:
Didn't work for me, I think i was getting unit test issues with that import pattern

Reply:
Really? The story works fine that way... 🤔 
Anyway, this is a minor detail we can adjust in a follow up PR to improve docs.